src/main/webapp/js/finscholar/singlepageview.js

```diff
@@ -29,7 +30,7 @@ class SinglePageView extends BasicView {
   constructor(dataHandler, template) {
     super();
     /**
-     * @private @const {!ScholarshipDataHandler} dataHandler_
+     * @private @const {!ScholarshipDataHandler | !CollegeDataHandler} dataHandler_
```

Comment:
I'll create a common datahandler today. You'll need to change the type to parent class later.